src/views/Swap/Swap.vue

```diff
@@ -1020,6 +1023,18 @@ export default {
     closeBridgeModal () {
       this.loading = false
       this.bridgeModalOpen = false
+    },
+    trackNoLiquidity () {
+      if (this.showNoLiquidityMessage) {
+        this.trackAnalytics({
+          event: 'No Liquidity',
+          properties: {
+            category: 'Swap screen',
+            action: 'No Liquidity for pairs',
+            label: `No Liquidity for ${this.asset} to ${this.toAsset}`
```

Comment:
@bradleySuira  can u change label to 

`fron:  ${this.asset}`,
`to: ${this.toAsset}`

Reply:
done mate